she_codes_news/news/models.py

```diff
@@ -1,8 +1,21 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+USER = get_user_model()
 
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+    # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        USER, on_delete=models.CASCADE
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    category = models.CharField(max_length=100, default="TBD")
```

Comment:
i would have used a many to many or a foreign key field to do categories 
you could also have used a choice field
i think using "free text" might end up with a messy site.